src/overlays/actors/ovl_En_Zl4/z_en_zl4.c

```diff
@@ -48,61 +61,1208 @@ const ActorInit En_Zl4_InitVars = {
     (ActorFunc)EnZl4_Update,
     (ActorFunc)EnZl4_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Zl4/func_80B5B7B0.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Zl4/func_80B5B888.s")
+ColliderCylinderInit sCylinderInit = {
+    { COLTYPE_UNK10, 0x00, 0x00, 0x39, 0x20, COLSHAPE_CYLINDER },
+    { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, 0x00, 0x00, 0x01 },
+    { 10, 44, 0, { 0, 0, 0 } },
+};
+
+CollisionCheckInfoInit2 sColChkInfoInit = { 0, 0, 0, 0, 0xFF };
+
+typedef enum {
+    /*  0 */ ZL4_ANIM_0,
+    /*  1 */ ZL4_ANIM_1,
+    /*  2 */ ZL4_ANIM_2,
+    /*  3 */ ZL4_ANIM_3,
+    /*  4 */ ZL4_ANIM_4,
+    /*  5 */ ZL4_ANIM_5,
+    /*  6 */ ZL4_ANIM_6,
+    /*  7 */ ZL4_ANIM_7,
+    /*  8 */ ZL4_ANIM_8,
+    /*  9 */ ZL4_ANIM_9,
+    /* 10 */ ZL4_ANIM_10,
+    /* 11 */ ZL4_ANIM_11,
+    /* 12 */ ZL4_ANIM_12,
+    /* 13 */ ZL4_ANIM_13,
+    /* 14 */ ZL4_ANIM_14,
+    /* 15 */ ZL4_ANIM_15,
+    /* 16 */ ZL4_ANIM_16,
+    /* 17 */ ZL4_ANIM_17,
+    /* 18 */ ZL4_ANIM_18,
+    /* 19 */ ZL4_ANIM_19,
+    /* 20 */ ZL4_ANIM_20,
+    /* 21 */ ZL4_ANIM_21,
+    /* 22 */ ZL4_ANIM_22,
+    /* 23 */ ZL4_ANIM_23,
+    /* 24 */ ZL4_ANIM_24,
+    /* 25 */ ZL4_ANIM_25,
+    /* 26 */ ZL4_ANIM_26,
+    /* 27 */ ZL4_ANIM_27,
+    /* 28 */ ZL4_ANIM_28,
+    /* 29 */ ZL4_ANIM_29,
+    /* 30 */ ZL4_ANIM_30,
+    /* 31 */ ZL4_ANIM_31,
+    /* 32 */ ZL4_ANIM_32,
+    /* 33 */ ZL4_ANIM_33
+} EnZl4Animation;
+
+struct_80034EC0_Entry sAnimationEntries[] = {
+    /*  0 */ /* stand */ { &gChildZeldaCYAnim_000654, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    /*  1 */ /* moves to introduce herself */ { &gChildZeldaCYAnim_00E5C8, 1.0f, 0.0f, -1.0f, ANIMMODE_ONCE, -1.0f },
+    /*  2 */ /* introducing herself */ { &gChildZeldaCYAnim_00EBC4, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -1.0f },
```

Comment:
comments should prob come after the data so that it stays a bit more in line. and im not sure that index comments in the data itself is necessary, we have the enum. the animations will be named eventually to correspond as well

Reply:
These were my notes while I tried to come up with names for them. As you can see, there's a lot of them and many are similar to each other. the numbers though are useful for the enum.